travellan/travellan-rn
# Feature/305 actionsheet on adding accommodation

## Changes made

- I removed a deprecated dependency and added a new, maintained library as a dependency - [react-native-actions-sheet](https://www.npmjs.com/package/react-native-actions-sheet),
- I created ActionSheet component for reusability (imagine a situation where we have to change the styling - it only happens in one place now, not throughout the application where ActionSheet is used).

## Screenshots

![actionsheet](https://user-images.githubusercontent.com/32842468/103768640-2a60da80-5023-11eb-9d78-a7dfb6a6f805.gif)

## Test path (if necessary)

1. Log in.
2. Choose any trip/create a new trip and select it from the list of trips.
3. Go to Accommodation.
4. Click on the 'plus' sign in the top right corner.
5. An action sheet should appear with clickable items. When you click on either one, you will be redirected to "Add accommodation" screen.

## Checklist

- [ ] merged develop branch into feature branch
  - [ ] in emulator
  - [ ] on a real device
- [ ] added new dependencies (what deps?)
- [ ] used ESLint (linted) and Prettier (formatted)
